app/Models/Signup.php

```diff
@@ -182,6 +182,18 @@ public function getQuantity()
         return $this->posts->sum('quantity');
     }
 
+    /**
+     * Get the quantity total associated with approved posts under this signup
+     *
+     * @return int
+     */
+    public function getAcceptedQuantity()
+    {
+        $accepted_posts = $this->posts->where('status', 'accepted');
```

Comment:
How's this look in terms of performance? (Specifically curious what kind of SQL queries this ends up running, and if it still performs well with production-sized data sets).

Reply:
good question! I'll check on it

Reply:
@DFurnes I used clockwork and my local Domino for this (since currently this optional include should only be used by Domino) and I'm clocking times in the ~700ms range but there are a bunch of queries. But are there too many queries? Of that, I am not sure! Want to swing by?

Reply:
Wowza, 91 queries?! That's a lot, but yeah doesn't seem related to this change.

Reply:
@DFurnes yup! there are more in the case when stuff is no longer in cache, but seems consistent whether or not `include=accepted_quantity` is used